### ISSUE TITLE
How to unhide a channel?

### ISSUE THREAD

**Issue by JohnW43** [PredatH0r/ChanSort#148]
In the .tll file I get from my LG TV some channels have their 'Hide' box ticked. How can I unhide them? Simply unticking the 'Hide' box does nothing: the channel is still invisible and the Channel number is blue. Also adding a name also does not help. 
Any suggestions, please?

**Comment by PredatH0r** [comment]
Normally unchecking "hide" should be sufficient.
Is this a channel that you hid and want to unhide now or was the channel already hidden after the channel search? In that case there might also be some other flags in the data file that may cause the TV to hide the channel.
If you can send me your channel list and the info which channel you want to unhide, I can take a look.

**Comment by Kyusetzu** [comment]
> Normally unchecking "hide" should be sufficient.
> Is this a channel that you hid and want to unhide now or was the channel already hidden after the channel search? In that case there might also be some other flags in the data file that may cause the TV to hide the channel.
> If you can send me your channel list and the info which channel you want to unhide, I can take a look.

Have you guys had any success?
Sorry to Necro this, but we're having the same issue with our LG TV.

**Comment by JohnW43** [comment]
> > Normally unchecking "hide" should be sufficient.
> > Is this a channel that you hid and want to unhide now or was the channel already hidden after the channel search? In that case there might also be some other flags in the data file that may cause the TV to hide the channel.
> > If you can send me your channel list and the info which channel you want to unhide, I can take a look.
> 
> Have you guys had any success?
> Sorry to Necro this, but we're having the same issue with our LG TV.

No, I did send the channel list (by e-mail) but never got a reply. So the problem still exists. Since you are having the same issue with the same LG TV it may be an issue with LG: I don't know.
Anyone found a solution yet?

**Comment by PredatH0r** [comment]
I can't recall having received any mail in this matter. Could be that the spam filter removed it or that I simply overlooked it. 
Can you please attach the file to this thread and tell me an example channel that is hidden and cannot be unhidden?

**Comment by JohnW43** [comment]
Please find a attached a .zip file which contains 2 channellists: one before and one after unhiding channel 801: BBC Parliament. The channel was already 'hidden' at the first channel-search on my LG-TV. Using your software ChanSort I unticked the 'hidden' status for this channel.
After copying the new list to the TV the channel number now shows up in blue but is still invisible.
Your advice is much appreciated.
[Channel lists.zip](https://github.com/PredatH0r/ChanSort/files/3936366/Channel.lists.zip)

**Comment by Kyusetzu** [comment]
Here is my channel list.
Number 11, VOX CH is constantly getting skipped when going through the channels.
After unhiding it appears 1x while skipping and gets automatically hidden again after that.
[GlobalClone00001.zip](https://github.com/PredatH0r/ChanSort/files/3970675/GlobalClone00001.zip)
